ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -230,68 +187,16 @@ bb_exit (void **pdata)
 {
   bounding_boxes *bdata = *pdata;
 
-  _free_labels (bdata);
+  _free_labels (&bdata->labeldata);
 
-  if (bdata->label_path)
-    g_free (bdata->label_path);
+  if (bdata->labeldata.label_path)
+    g_free (bdata->labeldata.label_path);
```

Comment:
How about moving this line inside `_free_labels`?

Reply:
No, we can't. Look at tensordecutil.c::35. label_path may be referred after calling it.

Reply:
Ah. I missed that. Got it.

Reply:
Arguably, `label_path` shouldn't be part of this `imglabel_t`

1. `label_path` is not highly coupled to the label itself, rather it is a way used to inject data. There might be other way to initiate `imglabel_t` later on.
2. `label_path` always has to be treated separately to `imglabel_t` as seen in tensordecutil.c::35 or here.

Reply:
@zhoonit A good point. Let's refactor it.

Reply:
Updated.